### DISCUSSION TITLE
Temporarily set log level and reset within a specific time

### DISCUSSION THREAD

**Discussion by morgul77** [datalust/seq-tickets#1429] [Ideas]
We have about 200 API keys, and using the ability to set the loglevel.

There are two main issues:
With 50 developers, I don't like to give them all admin rights to adjust the loglevel.
When setting the loglevel, it's easy to forget to reset it.

So, ideally (for us) non-admins could have the option to set the loglevel for a period of time, up to 24 hours, and it would revert to original setting.

I think this is ideal in OUR scenario, but I assume that forgetting to reset the log level is more common than the permissions issue.

**Comment by nblumhardt** [comment]
Thanks for the suggestion! We'll give this some more thought.
